network/core-node/admin-guide/publishing-history-archives.mdx

```diff
@@ -173,22 +156,25 @@ server {
 }
 ```
 
-</CodeExample>
+## Backfilling a History Archive
 
-## Backfilling a history archive
+Given the choice, it's best to configure your history archive prior to your node's initial sync with an existing network. That way your validator's history publishes as you join and subsequently sync with the network.
 
-Given the choice, it's best to configure your history archive prior to your node's initial synch to the network. That way your validator's history publishes as you join/synch to the network.
+However, if you have _not_ published an archive during the node's initial sync, it's still possible to use the [stellar-archivist](https://github.com/stellar/go/tree/master/tools/stellar-archivist) command line tool to mirror, scan, and repair existing archives. Using the [SDF package repositories](https://github.com/stellar/packages), you can install `stellar-archivist` by running:
```

Comment:
These paragraphs about using stellar-archivist seem to belong to the ##Complete History Archive section below.  Possible to consolidate?

Reply:
I've shuffled this around a bit, but I think you're right: it could definitely benefit from some re-thinking/consolidating. I'll let it marinade overnight, and see what I can make happen tomorrow.